project2/style.css

```
@@ -0,0 +1,232 @@
+/* TO CONTAIN AND BASE STYLE ALL ELEMENTS */
+
+body {
+    background-color: hsl(0, 0%, 85%);
+}
+
+/* taking the advice of leaving the body alone!! */
+.container {
+    background-color: hsl(0, 0%, 100%);
+    box-sizing: border-box;
+    font-family: 'Poppins', Verdana, Geneva, Tahoma, sans-serif;
+
+    font-size: 15px;
+    margin: auto;
+    padding: 20px;
+    position: relative;
+    width: 300px;
+    /* giving this PARENT a position of RELATIVE... (go to x34x) */
+}
+
+.heart {
+    color: hsl(4, 73%, 62%);
+    float: right;
+    font-size: 20px;
+    position: absolute;
+    /* x34x will allow this position of ABSOLUTE to make it absolute relative to that container, and not the entire BODY!! sweet*/
+    right: 25px;
+    top: 25px;
+    z-index: 999;
+}
+
+/* image area container */
+.imgandcart {
+    position: relative;
+    overflow: hidden;
+    /*Used to stop new height from making text overlap the color choice buttons and preventing their selection */
+    height: 380px;
+    /* HEIGHT must be definied because button still takes up space when not visible before transition animation. */
+}
+
+img {
+    max-width: 100%;
+}
+
+.cartbutton
+
+/*the button itself, not considering its hover stuff*/
+    {
+    background-color: hsla(177, 48%, 29%, .0);
+    color: hsl(37, 87%, 88%, 0);
+    display: block;
+    font-size: 20px;
+    font-weight: 600;
+    position: relative;
+    bottom: 40px;
+
+    width: 100%;
+    padding: 10px 0;
+    text-align: center;
+}
+
+.cartbutton p {
+    /* reset margins for this element.... maybe i should get rid of the over margin on the container for everything?*/
+    margin: 0;
+}
+
+/* individual information elements for product */
+
+.colorcontain {
+    height: 40px;
+    width: 100%;
+    padding: 0px 0;
```

This can be simplified and have the units stripped from it:
```suggestion
    padding: 0;
```